Backpack/src/main/java/net/skyscanner/backpack/text/internal/FontFamilyResolver.kt

```diff
@@ -0,0 +1,42 @@
+package net.skyscanner.backpack.text.internal
+
+import android.content.Context
+import android.graphics.Typeface
+import android.util.TypedValue
+import net.skyscanner.backpack.R
+import net.skyscanner.backpack.text.BpkText
+import net.skyscanner.backpack.text.FontCache
+
+class FontFamilyResolver(val context: Context) {
```

Comment:
@UIThread

Reply:
Not sure it applies anymore, but why is that?

Reply:
It's an indication what the elements is not a thread-safe and should be used in UI thread only. Since we want to make it a singleton with unsafe static initializations it makes sense.